### DISCUSSION TITLE
Inconsistency between Real Time Streaming Device Test Specification and Media Service Specification

### DISCUSSION THREAD

**Discussion by bigbrobrody** [onvif/testspecs#39] [General]
The Media Service Specification states that Get**Stream**Uri will include in the response InvalidAfterConnect, InvalidAfterReboot and Timeout parameters.

However, throughout the Real Time Streaming Device Test Specification it refers to ValidUntilConnect and ValidUntilReboot parameters in the context of GetStreamUri. The only exception to this is in the description of HelperGetStreamUri in appendix A.21, which seems to reference the correct parameters.

I note that the Media Service Specification states that ValidUntilConnect and ValidUntilReboot are parameters in the Get**Snapshot**Uri response.

**Comment by baldvin137** [comment]
Thanks for pointing this out. I think it should be **InvalidAfter** in all cases. A quick look on my phone and it looked like that the **ValidUntil** instances are only found in the documentation (both Media Specification and Test Specifications) and not in the wsdl. It is always the wsdl that is "the truth", also for the Test Tools. So I'm not expecting this to be a problem in the Test Tools, but it should definitely be corrected in both the Media Spec and the Test Specs.

**Comment by madhurao68** [comment]
The Real Time Streaming Device Test Specification is already updated.
